### PR TITLE
Fixed wrong default values in bastion.conf comments

### PR DESCRIPTION
**Pull request by axl89**
image

### CODE REVIEW

**Review by Alkorin — DISMISSED** [review]

**Comment by Alkorin** [comment]
Thanks!

**Review by Alkorin — DISMISSED** [review]

File: etc/bastion/bastion.conf.dist [hunk 1]
```diff
@@ -355,7 +355,7 @@
 #
 # ttyrecFilenameFormat (string)
 #    DESC: Sets the filename format of the output files of ttyrec for a given session. Magic tokens are: &bastionname, &uniqid, &account, &ip, &port, &user (they'll be replaced by the corresponding values of the current session). Then, this string (automatically prepended with the correct folder) will be passed to ttyrec's -F parameter, which uses strftime() to expand it, so the usual character conversions will be done (%Y for the year, %H for the hour, etc., see man strftime). Note that in a addition to the usual strftime() conversion specifications, ttyrec also supports #usec#, to be replaced by the current microsecond value of the time.
-# DEFAULT: %Y-%m-%d.%H-%M-%S.#usec#.&uniqid.ttyrec
+# DEFAULT: %Y-%m-%d.%H-%M-%S.#usec#.&uniqid.&account.&user.&ip.&port.ttyrec
```
Comment (speed47) [inline etc/bastion/bastion.conf.dist:358]: On this one however, I think the `dist` version is better (it's more precise), so let's apply the same value to `configuration.inc`:
```
$C->{'ttyrecFilenameFormat'} = '%Y-%m-%d.%H-%M-%S.#usec#.&uniqid.&account.&user.&ip.&port.ttyrec' if (not $C->{'ttyrecFilenameFormat'});
```

(around line 185)

**Review by speed47 — CHANGES_REQUESTED** [review]
Thanks for the PR!

File: etc/bastion/bastion.conf.dist [hunk 2]
```diff
@@ -88,7 +88,7 @@
 # ingressKeysFromAllowOverride (boolean-int, i.e. 0 or 1), aliases: ipWhiteListAllowOverride (deprecated)
 #    DESC: If set to 0 (false), any from="..." specified in user keys (selfAddIngressKey or accountCreate) are ignored and replaced by the IPs in the ingressKeysFrom configuration option (if any).
 #          If set to 1 (true), any from="..." specified in user keys (selfAddIngressKey or accountCreate) will override the value set in ingressKeysFrom (if any). When no user-specified from="..." appears, the value of ingressKeysFrom is still used, regardless of this option.
-# DEFAULT: 0
+# DEFAULT: 1
 "ingressKeysFromAllowOverride": 1,
```
[the remark this answers is not in the document]
Comment (axl89) [reply etc/bastion/bastion.conf.dist]: Ah, I completely misunderstood it. Thank you for the explanation! :smile:

File: etc/bastion/bastion.conf.dist [hunk 3]
```diff
@@ -135,7 +135,7 @@
 #
 # minimumIngressRsaKeySize (int), deprecated alias: minimumRsaKeySize
 #     DESC: The minimum allowed size for ingress RSA keys (user->bastion). Sane values range from 2048 to 4096.
-#  DEFAULT: 2048
+#  DEFAULT: 4096
```
[the remark this answers is not in the document]
Comment (axl89) [reply etc/bastion/bastion.conf.dist]: Changed in 534ce45812802d1760c095d1a82d87f193698987

File: etc/bastion/bastion.conf.dist [hunk 4]
```diff
@@ -145,7 +145,7 @@
 #
 # minimumEgressRsaKeySize (int), deprecated alias: minimumRsaKeySize
 #     DESC: The minimum allowed size for egress RSA keys (bastion->server). Sane values range from 2048 to 4096.
-#  DEFAULT: 2048
+#  DEFAULT: 4096
```
[the remark this answers is not in the document]
Comment (axl89) [reply etc/bastion/bastion.conf.dist]: Changed in 534ce45812802d1760c095d1a82d87f193698987

File: etc/bastion/bastion.conf.dist (hunk 1 above)
Comment (axl89) [reply etc/bastion/bastion.conf.dist:358]: Changed in 9b06db1461ba91932cff469a9169ad0f309405af

**Review by speed47 — APPROVED** [review]
Thanks! As this changes some default parameters, we'll merge this on when we're on the way to release `v3.02.00`, just in case we have to release a `v3.01.02` before it!

**Review by deathiop — APPROVED** [review]

**Review by Alkorin — APPROVED** [review]

**Pull request merged**